src/components/layout/nav/search/Search.jsx

```diff
@@ -1,7 +1,156 @@
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332762662_10_a-Nike-Womens-Air-Precision-II-Basketball-Shoes-White-Black-AA7069-100?bgColor=0,0,0,0&fmt=png-alpha&hei=528&resMode=sharp&qlt=85,1&op_sharpen=1",
+    ],
+  },
+  {
+    title: "Airmax Z500",
+    brand: "Nike",
+    price: 99,
+    currency: "$",
+    tags: ["shoe", "shoes", "nike", "airmax", "air"],
+    images: [
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332979914_01_a?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332654173_01_a-Nike-Mens-Flex-Control-3-Training-Shoes-Black-Anthracite-AJ5911-002?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332762662_10_a-Nike-Womens-Air-Precision-II-Basketball-Shoes-White-Black-AA7069-100?bgColor=0,0,0,0&fmt=png-alpha&hei=528&resMode=sharp&qlt=85,1&op_sharpen=1",
+    ],
+  },
+  {
+    title: "YEEZY BOOST 350",
+    brand: "Adidas",
+    price: 399,
+    currency: "$",
+    tags: ["shoe", "shoes", "adidas", "yeezy", "boost"],
+    images: [
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332979914_01_a?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332654173_01_a-Nike-Mens-Flex-Control-3-Training-Shoes-Black-Anthracite-AJ5911-002?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332762662_10_a-Nike-Womens-Air-Precision-II-Basketball-Shoes-White-Black-AA7069-100?bgColor=0,0,0,0&fmt=png-alpha&hei=528&resMode=sharp&qlt=85,1&op_sharpen=1",
+    ],
+  },
+  {
+    title: "V2 ZYON",
+    brand: "Adidas",
+    price: 250,
+    currency: "$",
+    tags: ["shoe", "shoes", "adidas", "v2", "zyon"],
+    images: [
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332979914_01_a?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332654173_01_a-Nike-Mens-Flex-Control-3-Training-Shoes-Black-Anthracite-AJ5911-002?wid=800&hei=800&bgColor=0,0,0,0&fmt=png-alpha&resMode=sharp2&op_sharpen=1",
+      "https://fgl.scene7.com/is/image/FGLSportsLtd/FGL_332762662_10_a-Nike-Womens-Air-Precision-II-Basketball-Shoes-White-Black-AA7069-100?bgColor=0,0,0,0&fmt=png-alpha&hei=528&resMode=sharp&qlt=85,1&op_sharpen=1",
+    ],
+  },
+];
 
 const Search = () => {
-  return <div></div>;
+  // It is left empty because we want search to start as empty.
+  const [searchQueryContainer, setSearchQuery] = useState("");
+  const [showEmptySearchWarning, setEmptySearchWarning] = useState(false);
+
+  const onlyFirstImageOfProduct = 0;
+  const amountOfProductToShowInSearch = 3;
+
+  // This will be sending input value to database with fetching if input is not empty when "Search" button is clicked.
```

Comment:
generally we do this:

```
// TODO(mygithubname): Link this to the database
```

this makes it clear that this is code still left to be done. i think this code should link to the backend eventually right? you can still keep your comment above though, it's good